REQUIRE

```diff
@@ -1,7 +1,4 @@
 julia 0.5-
 Lazy
-LNR
-JuliaParser
```

Comment:
I guess these are vestiges from when we did scoping here, so it should be safe to remove them if they're not obviously used.